src/containers/Profile/ViewProfile.js

```diff
@@ -16,17 +17,6 @@ import './Profile.scss';
  * @extends {Component}
  */
 class ViewProfile extends Component {
-  /**
```

Comment:
I moved this part of the code to Header.js because both CreateArticle.js and ViewProfile.js need the same profile data.